coops/titans_stronghold.tex

```diff
@@ -96,6 +92,7 @@ \subsection*{\MakeUppercase{Additional Rules}}
 
   \item Whenever a player Visits an Obelisk, that player rolls one treasure Die and one resource Die, and resolves one of them.
   \item When all VII Fields (excluding the Titans' Stronghold) are flagged, randomly draw and shuffle the specified number (see the next page) of Neutral Unit Cards from each of their corresponding Decks to create a separate Deck of Neutral Units for the Titans' Stronghold (the Deck of the Titans' Stronghold is sometimes split in two because certain Scenarios are otherwise impossible to implement with the Cards from certain expansions).
+  \vspace*{\fill}\columnbreak
```

Comment:
```suggestion
```
This one is not necessary. It doesn't improve the reading experience or the page layout. It only makes the page have a blank stripe at the bottom.